app/store/modules/product/index.js

```diff
@@ -1,25 +1,78 @@
 // store/modules/product/index.js
 
 import productData from './mock/product';
+import api from './../../utils/api';
 
 export default {
   namespaced: true,
   state: {
     /**
      * @type {Object}   Stores product data
      */
-    product: productData
+    product: productData,
+
+    /**
+     * @type {Object}   Stores product information data
+     */
+    productInformation: {},
+
+    /**
+     * @type {Object} API error
+     */
+    apiError: null,
   },
   getters: {
     /**
      * Gets a product
      *
      * @param   {Object}  state   Current state
      *
-     * @returns  {Array}  product   Product data
+     * @returns  {Object}  product   Product data
+     */
+    getProduct: state => state.product,
+
+    /**
+     * Get the product information
+     *
+     * @param   {Object}  state   Current state
+     *
+     * @returns  {Object}  product   Product information data
+     */
+    getProductInformation: state => state.productInformation,
+  },
+  mutations: {
+    /**
+     * Sets the productinformation for the current state
+     * @param {Object} state Current state
+     * @param {Object} productInformation Product information data
      */
-    getProduct: state => state.product
+    setProductInformation(state, productInformation) {
+      state.productInformation = productInformation;
+    },
+
+    /**
+     * Handles an API failure
+     * @param {Object} state Current state
+     * @param {Object} error API Error that ocured
+     */
+    apiFailure(state, error) {
+      state.apiError = error;
+    }
+  },
+  actions: {
+    getProductInformation({ commit, state }) {
+      const payload = [{
+        sku: state.product.sku,
+        quantity: state.product.quantity
+      }];
+
+      return api.post('/product/multi-get', payload)
+        .then((response) => {
+          if (response && response instanceof Array && response.length > 0) {
```

Comment:
- [ ] Please use `Array.isArray(response) && response.length` instead. `length` is always a number if existing and `0` is falsy in JS. You don't need to check `> 0`.

Reply:
- [ ] Please note that the response data is always inside a `data` attribute. So it's `response.data`. See https://winterhalter-fenner.atlassian.net/wiki/spaces/WF/pages/241664001/General+structure